pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1471,9 +1483,11 @@ func PublishEdgeviewToZedCloud(ctx *zedagentContext, evStatus *types.EdgeviewSta
 	//We queue the message and then get the highest priority message to send.
 	//If there are no failures and defers we'll send this message,
 	//but if there is a queue we'll retry sending the highest priority message.
-	zedcloud.SetDeferred(zedcloudCtx, "global", buf, size, statusURL,
+	const ignoreErr = LOCDest
+	ditems := createDeferredItemsByDest(ctx, dest, ignoreErr, "global", buf, size,
 		true, false, info.ZInfoTypes_ZiEdgeview)
-	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, true)
+	zedcloud.AddDeferredList(zedcloudCtx, ditems)
+	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, false)
```

Comment:
Why are you changing the last argument to HandleDeferred from true to false here?